apps/strapi-cms/Dockerfile

```diff
@@ -28,4 +28,5 @@ ENV PATH /opt/node_modules/.bin:$PATH
 RUN chown -R node:node /opt/app
 USER node
 EXPOSE 1337
+RUN npm run configrestore
```

Comment:
How has this been tested?

Reply:
As far as functionality goes it's been tested locally during PR #81